docs/README-release.md

```diff
@@ -0,0 +1,28 @@
+README-release.md
+========
+
+Conventions for Caffeine Releases
+-------------
+1. Nominate a Release Manager with primary responsibility for ensuring each step in this
+   procedure is followed
+2. Ensure there are no open issues marked with `complete-before-release` label
+3. Validate correctness testing has been performed across all supported systems and supported
+   versions of external dependencies
+4. Update all instances of the copyright year embedded in: [LICENSE.txt](../LICENSE.txt),
+   [fpm.toml (created by install.sh)](../install.sh)
+5. Update all instances of the release package version number embedded in: [fpm.toml (created by install.sh)](../install.sh),
+   [install.sh](../install.sh)
+6. Update the author list embedded in: [fpm.toml(created by install.sh)](../install.sh)
```

Comment:
I think most of the toml inputs are actually need to be updated in manifest/fpm.toml.template (which is read by install)

```suggestion
4. Update all instances of the copyright year embedded in: [LICENSE.txt](../LICENSE.txt),
   [manifest/fpm.toml.template](../manifest/fpm.toml.template)
5. Update all instances of the release package version number embedded in: [manifest/fpm.toml.template](../manifest/fpm.toml.template) and
   [install.sh](../install.sh)
6. Update the author list embedded in: [manifest/fpm.toml.template](../manifest/fpm.toml.template)
```

Reply:
Great catch @bonachea! Thanks!